src/components/NodeMovingPlaceholder/index.test.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import NodeMovingPlaceholder from './index';
+
+const svg = document.createElement('svg');
+
+const defaultProps = {
+  node: {
+    id: 'node-id',
+    states: [],
+    parents: [],
+    position: { x: 0, y: 0 },
+    size: { width: 160, height: 100 },
+  },
+  svg,
+  onSetPosition: jest.fn(),
+  onCancel: jest.fn(),
+};
+
+const shallowComponent = (props = {}) => {
+  const compProps = { ...defaultProps, ...props };
+
+  return shallow(<NodeMovingPlaceholder {...compProps} />);
+};
+
+describe('NodeMovingPlaceholder Component', () => {
+  describe('Event Listeners', () => {
+    describe('Check bind and unbind events', () => {
+      beforeEach(() => {
+        svg.addEventListener = jest.fn();
```

Comment:
Can this object be recreated for every test as well? It can be put inside the same `beforeEach`. Avoiding shared state between tests is really important.